### PR TITLE
React `18.3.3`

### PR DESCRIPTION
**Pull request by porotkin**
cc @turansky, @aerialist7

### CODE REVIEW

File: react-kotlin/src/jsMain/kotlin/react/dom/html/ImgFetchPriority.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Automatically generated - do not modify!
+
+package react.dom.html
+
+import seskar.js.JsValue
+
+sealed external interface ImgFetchPriority {
```
Comment (turansky) [inline react-kotlin/src/jsMain/kotlin/react/dom/html/ImgFetchPriority.kt:7]: Redundant?

File: buildSrc/src/main/kotlin/karakum/react/MemberConverter.kt [hunk 2]
```diff
@@ -78,6 +86,7 @@ private fun convertProperty(
 
     val sourceType = source.substringAfter(": ")
         .replace("EventTarget & T", "T")
+        .removePrefix("   | ") // TODO: Check after multiline unions patch
```
Comment (turansky) [inline buildSrc/src/main/kotlin/karakum/react/MemberConverter.kt:89]: Text  patch before conversion?

**Review by turansky — CHANGES_REQUESTED** [review]

File: react-kotlin/src/jsMain/kotlin/react/dom/html/ImgFetchPriority.kt (hunk 1 above)
Comment (porotkin) [reply react-kotlin/src/jsMain/kotlin/react/dom/html/ImgFetchPriority.kt]: yep

File: buildSrc/src/main/kotlin/karakum/react/MemberConverter.kt (hunk 2 above)
Comment (porotkin) [reply buildSrc/src/main/kotlin/karakum/react/MemberConverter.kt]: done

File: buildSrc/src/main/kotlin/karakum/common/Strings.kt [hunk 3]
```diff
@@ -28,3 +30,6 @@ internal fun String.replaceSuffix(
 
     return removeSuffix(from) + to
 }
+
+internal fun String.removeQuoteSurrounding(): String = removeSurrounding("'")
+    .removeSurrounding("$quote")
```
Comment (porotkin) [inline buildSrc/src/main/kotlin/karakum/common/Strings.kt:35]: "\""

File: buildSrc/src/main/kotlin/karakum/react/AutoComplete.kt [hunk 4]
```diff
@@ -1,60 +1,5 @@
 package karakum.react
 
 internal const val AUTO_COMPLETE = """
```
Comment (turansky) [inline buildSrc/src/main/kotlin/karakum/react/AutoComplete.kt:3]: Is it used?

**Review by turansky — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/kotlin/karakum/react/AutoComplete.kt (hunk 4 above)
Comment (porotkin) [reply buildSrc/src/main/kotlin/karakum/react/AutoComplete.kt:3]: yes

File: buildSrc/src/main/kotlin/karakum/react/UnionConverter.kt [hunk 5]
```diff
@@ -37,3 +39,18 @@ internal fun convertUnion(
         name = name,
         body = sealedUnionBody(name, values),
     )
+
+internal fun String.withNormalizedUnions(): String {
```
[the remark this answers is not in the document]
Comment (porotkin) [reply buildSrc/src/main/kotlin/karakum/react/UnionConverter.kt]: no such thing anymore, outdated concern

**Review by turansky — APPROVED** [review]

**Pull request merged**